aws/resource_aws_waf_web_acl.go

```diff
@@ -140,12 +140,14 @@ func resourceAwsWafWebAcl() *schema.Resource {
 					},
 				},
 			},
+			"tags": tagsSchema(),
 		},
 	}
 }
 
 func resourceAwsWafWebAclCreate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).wafconn
+	tags := tagsFromMapWAF(d.Get("tags").(map[string]interface{}))
```

Comment:
After adding the import `"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"` we can use the following:

```suggestion
	tags := keyvaluetags.New(d.Get("tags").(map[string]interface{})).IgnoreAws().WafTags()
```